examples/encoding/drm_integrations/create_drm_encoding_with_verimatrix.py

```diff
@@ -0,0 +1,402 @@
+    VideoAdaptationSet, AudioAdaptationSet, ContentProtection, HTTPSInput, HlsManifest, VariantStream, \
+    AudioMedia, FairPlayDRM, TSMuxing, AWSCloudRegion
+from bitmovin import CENCDRM as CENCDRMResource
+from bitmovin.resources.models import CENCPlayReadyEntry, CENCWidevineEntry
+from bitmovin.errors import BitmovinError
+
+def content_request(endpoint, contentId, siteId):
+    contentURL = urllib.parse.urljoin(endpoint, 'cei/v1.0/content/{}?siteId={}')
+    contentURL = contentURL.format(contentId, siteId)
+
+    contentRequest = """{
+        "contentRequest": {
+            "siteId": "",
+            "contentId": "",
+            "contentType": "VOD",
+            "streamingProtocol": "DASH",
+            "encryptionAlgorithm": "AES_CTR",
+            "keyRotation": {
+                "chainedLicense": "FALSE",
+                "cryptoPeriod": "0"
+            }
+        }
+    }"""
+
+    parsed_json = json.loads(contentRequest)
+    parsed_json["contentRequest"]["contentId"] = contentId
+    parsed_json["contentRequest"]["siteId"] = siteId
+
+    headers = {'Content-Type': 'application/json'}
+
+    r = requests.put(contentURL, data=json.dumps(parsed_json),headers=headers)
+    if r.status_code != 200:
+        print(r.status_code)
+        print("Did not get 200 back for content request")
+        exit()
+    parsed_json = json.loads(r.text)
+    if parsed_json["contentResponse"]["statusCode"] != "OPERATION_SUCCESS":
+        print(parsed_json["contentResponse"]["statusCode"])
+        print("Did not get OPERATION_SUCCESS for content request")
+        exit()
+
+def key_request(endpoint, contentId, siteId):
+    keyURL = urllib.parse.urljoin(endpoint, 'cei/v1.0/content/{}/position/0/key?siteId={}')
+    keyURL = keyURL.format(contentId, siteId)
+
+    keyRequest = """{
+        "keyRequest": {
+            "contentInfo": {
+                "contentId": "igetreplaced",
+                "siteId": "",
+                "keyCount": "1",
+                "contentKeyList": [{
+                    "keyId": "",
+                    "key": "",
+                    "position": "0"
+                }]
+            }
+        }
+    }"""
+    parsed_json = json.loads(keyRequest)
+    parsed_json["keyRequest"]["contentInfo"]["contentId"] = contentId
+    parsed_json["keyRequest"]["contentInfo"]["siteId"] = siteId
+
+    headers = {'Content-Type': 'application/json'}
+
+    r = requests.put(keyURL, data=json.dumps(parsed_json),headers=headers)
+    if r.status_code != 200:
+        print(r.status_code)
+        print("Did not get 200 back for key request")
+        exit()
+    parsed_json = json.loads(r.text)
+    if parsed_json["keyResponse"]["statusCode"] != "OPERATION_SUCCESS":
+        print(parsed_json["keyResponse"]["statusCode"])
+        print("Did not get OPERATION_SUCCESS for key request")
+        exit()
+
+    foundCENCPlayready = False
+    foundCENCWidevine = False
+
+    CENC_KEY = ''
```

Comment:
Isn't the general convention in python to have underscores and lowercase variable names? `cenc_key`, `cenc_kid` etc.. since you return those at the end of the function. The uppercase casing apparently denotes constants.. https://www.python.org/dev/peps/pep-0008/#constants